### ISSUE TITLE
Invalid version string in composer.json

### ISSUE THREAD

**Issue by schams-net** [DavidBruchmann/typo3_forum#29]
PHP composer reports an invalid version string `"^8.7.*"` in file `composer.json`.
```
$ composer config repositories.github vcs https://github.com/DavidBruchmann/typo3_forum
$ composer require mittwald/typo3_forum "dev-develop"
Reading composer.json of mittwald/typo3_forum (develop)                
Skipped branch develop, Could not parse version constraint ^8.7.*: Invalid version string "^8.7.*"

[InvalidArgumentException]
Could not find package mittwald/typo3_forum in a version matching dev-develop
```

**Comment by DavidBruchmann** [comment]
primary it's about `"name": "mittwald/typo3_forum",`.

`"typo3/cms-core": "^8.7.*",` simply should be changed to `"typo3/cms-core": "^8.7",` probably

**Comment by DavidBruchmann** [comment]
related: #5

**Comment by DavidBruchmann** [comment]
Solved with 67ac8e4, 90e68a9 and 00f72d8
